code-challenges.js

```diff
@@ -1,10 +1,26 @@
 // ASSESSMENT 3: Coding Practical Questions
 
 
-// --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+// --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. 
+//Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+//had to get Andee's help on this... felt like the logic I had for it kept not working and it made me mad. Not sure how to make this work without an input, although the prompt didn't specify WHAT the input should be. Need help understanding it more on Monday.
```

Comment:
good use of resources asking for help! in this case, there is not an input. it simply calculates a value when the function is called. but it _could_ be refactored to take any number an an input, and return that many elements in the fibonacci sequence